buildSrc/src/main/java/Config.kt

```diff
@@ -104,7 +104,7 @@ object Config {
         val retrofit2 = "$retrofit2Group:retrofit:$retrofit2Version"
         val retrofit2Gson = "$retrofit2Group:converter-gson:$retrofit2Version"
 
-        val coroutinesCore = "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.3"
```

Comment:
I presume our integration now only works from `1.6.0` onwards since it uses `CopyableThreadContextElement`?

Reply:
Hmm, correct, should we use the lowest supported version number here? I.e. 1.6.0 instead of 1.7.1 ?

Reply:
Updated https://github.com/getsentry/sentry-android-gradle-plugin/pull/521